app/helpers/application_helper.rb

```diff
@@ -32,4 +32,9 @@ def assocation_empty?(assoc)
     assoc.reload unless assoc.nil? || assoc.loaded?
     assoc.nil? ? true : assoc.size == 0
   end
+
+  def i18n_time_ago_in_words(past_time)
+    "#{time_ago_in_words(past_time)} #{t('time_ago')}"
```

Comment:
Good idea, thesuss: I'll fix this, then we can do a separate PR for the the translations.  (We'll be able to keep going and it'll hopefully be easy for you to just focus on the translation stuff when it's the only stuff in the PR.)